Backend/src/main/java/com/oops/backend/Backend/models/SolverBody.java

```diff
@@ -0,0 +1,34 @@
+package com.oops.backend.Backend.models;
+
+public class SolverBody {
+    private String solverName;
```

Comment:
I dont now if this is relevant with converting from Json to model, but api spec changed to have the first letter capitalized.